src/actions/entries.js

```diff
@@ -310,7 +310,7 @@ export function deleteEntry(collection, slug) {
     dispatch(entryDeleting(collection, slug));
     return backend.deleteEntry(state.config, collection, slug)
     .then(() => {
-      return dispatch(entryDeleted(collection, slug));
+      return Promise.resolve(dispatch(entryDeleted(collection, slug)));
```

Comment:
I don't think the `Promise.resolve` here isn't doing anything - `dispatch(entryDeleted(...))` should already be returning a promise.